### PR TITLE
[tests-only] wait for open file action menu to be closed before trying to open it again

### PR DESCRIPTION
**Pull request by kiranparajuli589**
Signed-off-by: Kiran Parajuli <kiranparajuli589@gmail.com>

## Description

Sometime, the open file actions menu comes in a way while the tests tries to click the `3 dot` button to open file actions menu (for some other resource).

Due to this sometimes we see failures like reported here in https://github.com/owncloud/encryption/issues/333

With this PR, we wait for the open actions menu to not be visible on the screen before attempting a click to open actions menu for some another resource.

## Related Issue

- https://github.com/owncloud/encryption/issues/333

## How Has This Been Tested?

- locally
- :robot:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by SwikritiT** [comment]
@kiranparajuli589  Ci is failing in `coding-standard`

**Comment by kiranparajuli589** [comment]
> @kiranparajuli589 Ci is failing in `coding-standard`

fixed!

File: tests/acceptance/features/lib/FilesPageBasic.php [hunk 1]
```diff
@@ -56,6 +56,7 @@ abstract class FilesPageBasic extends OwncloudPage {
 	protected $detailsDialogXpath = "//*[contains(@id, 'app-sidebar') and not(contains(@class, 'disappear'))]";
 	protected $newFileFolderButtonXpath = './/*[@id="controls"]//a[@class="button new"]';
 	protected $fileActionsApplicationSelectMenuXpath = ".//div[contains(@class, 'fileActionsApplicationSelectMenu')]";
+	protected $fileActionMenuCss = ".fileActionsMenu.open";
```
Comment (SwikritiT) [inline tests/acceptance/features/lib/FilesPageBasic.php:59]:
```suggestion
	protected $fileActionMenuOpenCss = ".fileActionsMenu.open";
```
Maybe?

**Review by SwikritiT — COMMENTED** [review]
otherwise looks good to me

**Review by kiranparajuli589 — COMMENTED** [review]
.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**